lib/api/CDataFrameTrainBoostedTreeRunner.cc

```diff
@@ -152,9 +154,14 @@ CDataFrameTrainBoostedTreeRunner::CDataFrameTrainBoostedTreeRunner(
     if (softTreeDepthTolerance > 0.0) {
         m_BoostedTreeFactory->softTreeDepthTolerance(softTreeDepthTolerance);
     }
+    // This is deprecated in favour of max_trees
+    // Will be removed after java side is changed to pass max_trees
```

Comment:
I think it is worth marking as a TODO
```suggestion
    // TODO Will be removed after java side is changed to pass max_trees
```